### PR TITLE
Add transform fot the Memphis Restaurants and Menus collection.

### PR DESCRIPTION
**Pull request by mlhale7**
## What does this Pull Request do?

This PR adds XSLT to transform p16108coll5 (Restaurant and Menu Collection) from Memphis Public Libraries. OAI XML for testing is included [here](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/files/5289069/p16108coll5.txt).

## How should this be tested?

A description of what steps someone could take to:

* Does the transform validate in Oxygen using the Saxon 8.7 processor
* Using sample XML code, does the pull request do what is intended
* If you run a full Schema validation on the feed the transform(s) applies to(using Variety.js and DLTN Metadata QA), does every record have:
	* a titleInfo/title
	* a location/url
	* an accessCondition

## Additional Notes:

I ran the contentdm_catalog_manifest_generator agains this set and it looks like none of the resources in this collection have a manifest. Because of this, I have not included any code in the transform to create a IIIF manifest url. Ultimately it would be best to review all of the Memphis public sets and create a single transform. For now I've written this set specific transform so that we can get the records (over 1000) in.

## Interested parties

@markpbaggett

### CODE REVIEW

**Review by markpbaggett — COMMENTED** [review]
@mlhale7, I'm still reviewing and testing, but wanted to point our this:

```
Error on line 178 of file:/home/mark/metadata/DLTN_XSLT/XSLT/memphisp16108coll5dctomods.xsl:
  XTTE0520: Required item type of @select attribute of xsl:apply-templates is node();
  supplied value has item type xs:string
Failed to compile stylesheet. 1 error detected.
```

File: XSLT/memphisp16108coll5dctomods.xsl [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    <!-- subject(s) -->
+    <!-- for subjects, whether they contains a ';' or not -->
+    <xsl:template match="dc:subject">
+        <xsl:variable name="subj-tokens" select="tokenize(., '; ')"/>
+        <xsl:for-each select="$subj-tokens">
+            <xsl:choose>
+                 <xsl:when test="ends-with(., ';')">
+                    <subject>
+                        <topic>
+                            <xsl:apply-templates select="substring(., 1, string-length(.) -1)"/>
```
Comment (markpbaggett) [inline XSLT/memphisp16108coll5dctomods.xsl:178]: @mlhale7 We're using the wrong xsl function here.  Like on 184, we need to us value-of:

```
<xsl:value-of select="substring(., 1, string-length(.) -1)"/>
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]

**Comment by markpbaggett** [comment]
@mlhale7, one more thing. Could you add a test to `tests/validate_scenarios.sh`?

In this file, around line 110, could you paste this **BEFORE** `. shunit2`:

```
# Test New Memphis XSLT
testValidityOfNewMemphisPublic(){
    curl "http://dpla.lib.utk.edu/repox/OAIHandler?verb=ListRecords&set=memphispublic_p16108coll5&metadataPrefix=oai_dc" 2>&1 2>/dev/null 1>"temp_memphis.xml"
    ${SAXON} delete.xml ${STYLESHEETS}/memphisp16108coll5dctomods.xsl 2>&1 2>/dev/null 1>${TESTFILE}
    RESPONSE=$(xmllint --noout --schema ${DLTNMODS} ${TESTFILE} 2>&1 1>/dev/null | cat)
    assertEquals "${RESPONSE}" "${TESTFILE} validates"
}

```

File: XSLT/memphisp16108coll5dctomods.xsl (hunk 1 above)
Comment (mlhale7) [reply XSLT/memphisp16108coll5dctomods.xsl:178]: Thanks for catching this!

**Comment by mlhale7** [comment]
@markpbaggett - I've incorporated the requested changes. Let me know if anything else needs editing.

File: XSLT/memphisp16108coll5dctomods.xsl (hunk 1 above)
Comment (markpbaggett) [reply XSLT/memphisp16108coll5dctomods.xsl]: Looks good.

**Review by markpbaggett — APPROVED** [review]
:+1:

**Pull request merged**